packages/sdk/python/human-protocol-sdk/human_protocol_sdk/job.py

```diff
@@ -651,16 +671,16 @@ def bulk_payout(
             "hmt_server_addr": self.hmt_server_addr,
         }
 
-        hash_, url = upload(
-            msg=results,
-            public_key=pub_key,
-            encrypt_data=encrypt_final_results,
-            use_public_bucket=store_pub_final_results,
+        hash_ = self.storage.upload_files([results], ESCROW_BUCKETNAME)[0]
+        url = (
+            ("https://" if ESCROW_SECURE else "http://")
+            + ESCROW_ENDPOINT_URL
```

Comment:
URLS include https and stuff. Perhaps this is a hostname name a URL? Any reason why we don't just assume the url starts with http something or other, and add https:/// if it doesn't exist rather than an escrow secure variable?